ibverbs/README.md

```diff
@@ -0,0 +1,67 @@
+# ibverbs
```

Comment:
We probably still want a top-level `README` as well. Not sure how other projects go about this? Maybe symlink one to the other?

Reply:
Let me look around how projects with workspaces handle this. IIRC they usually just have identical readme's on the root and main crate, but a symlink might be nicer.